openedx/features/course_experience/__init__.py

```diff
@@ -1,8 +1,34 @@
-# Unified course experience settings.
+"""
+Unified course experience settings and helper methods.
+"""
+
+import waffle
+
+from request_cache.middleware import RequestCache
 
 # Waffle flag to enable a single unified "Course" tab.
 UNIFIED_COURSE_EXPERIENCE_FLAG = 'unified_course_experience'
 
 # Waffle flag to enable the full screen course content view
 # along with a unified course home page.
 UNIFIED_COURSE_VIEW_FLAG = 'unified_course_view'
+
+
+def main_course_url_name(request=None):
+    """
+    Returns the main course URL for the current user.
```

Comment:
1. Can we change `main` to `default` in the name and comment?  
2. Could we drop `for the current user` from the comment, which will only live as long as the waffle flag.
3. Can we add `name` to the comment.
In other words:
```
Returns the default course URL name.
```